src/index.html

```diff
@@ -7,8 +7,79 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+  <header class="header">
+    <a class="logo" href="home">
+      <img class="image"
+        src="images/logo.png"
+        alt="logo"
+      >
+    </a>
+    <nav class="nav">
+      <ul class="list">
+        <li class="list-item">
+          <a href="Apple" class="link is-active"
+          >
+            Apple
+          </a>
+        </li>
+
+        <li class="list-item">
+          <a href="Samsung" class="link"
+          >
+            Samsung
+          </a>
+        </li>
+
+        <li class="list-item">
+          <a href="Smartphones" class="link"
+          >
+            Smartphones
+          </a>
+        </li>
+
+        <li class="list-item">
+          <a href="Laptops & computers"
+            class="link"
+            data-qa="hover"
+          >
+            Laptops & computers
+          </a>
+        </li>
+
+        <li class="list-item">
+          <a href="gadgets" class="link"
+          >
```

Comment:
```suggestion
          <a href="gadgets" class="link">
```
fix it all over your code